From dcf58b3471b1c9ba87a826aa132033e506664808 Mon Sep 17 00:00:00 2001
From: Werner Koch <wk@gnupg.org>
Date: Wed, 6 Aug 2014 17:09:15 +0200
Subject: [PATCH 1/2] Add kbnode_t for easier backporting.

* g10/global.h (kbnode_t): New.
---
 g10/global.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/g10/global.h b/g10/global.h
index 643c4fc..5ecfb83 100644
--- a/g10/global.h
+++ b/g10/global.h
@@ -23,6 +23,7 @@
 #define MAX_FINGERPRINT_LEN 20
 
 typedef struct kbnode_struct *KBNODE;
+typedef struct kbnode_struct *kbnode_t;
 typedef struct keydb_search_desc KEYDB_SEARCH_DESC;
 
 #endif /*GPG_GLOBAL_H*/
-- 
2.1.0

